apps/mark-scan/backend/src/custom-paper-handler/cli/state_machine_cli.ts

```diff
@@ -1,129 +0,0 @@
-/* istanbul ignore next */
```

Comment:
This CLI is the only caller of `apps/mark-scan/backend/src/custom-paper-handler/fixtures/ballot-pdf-data.bin`. It doesn't add value since the state machine is more complicated now, but it requires occasional maintenance when types etc. change. Since we have to maintain it but get no value from it we should just delete it.